features/draft/srcset.yml

```diff
@@ -0,0 +1,15 @@
+draft_date: 2024-07-17
```

Comment:
A comment here on why we didn't immediately publish would be nice to have

Reply:
I've added a comment up top.